src/sentry/static/sentry/app/components/projects/bookmarkToggle.jsx

```diff
@@ -1,37 +1,61 @@
-import PropTypes from 'prop-types';
 import React from 'react';
+import Reflux from 'reflux';
+import classNames from 'classnames';
 
 import createReactClass from 'create-react-class';
 
 import ApiMixin from '../../mixins/apiMixin';
 
 import {update as projectUpdate} from '../../actionCreators/projects';
 
+import LatestContextStore from '../../stores/latestContextStore';
+
 const BookmarkToggle = createReactClass({
```

Comment:
maybe leave a comment that this is coupled to the current project now? the implementation looks good